oca_dependencies.txt

```diff
@@ -0,0 +1,2 @@
+# Until merged
+knowledge https://github.com/Eficent/knowledge 12.0-mig-knowledge
```

Comment:
This is not working with the same branch. You have to tweak the .travis.yml instead

Reply:
how?

Reply:
Putting `git clone` command for this branch and downloading in a different directory, and moving (`mv` command) the `knowledge` directory to the main repository one.

Reply:
I tried but seems I did something wrong. Can you check?